static/app/components/workflowEngine/gridCell/monitorsCell.tsx

```diff
@@ -0,0 +1,125 @@
+import {Fragment} from 'react';
+import {css} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import {Hovercard} from 'sentry/components/hovercard';
+import ProjectBadge from 'sentry/components/idBadge/projectBadge';
+import Link from 'sentry/components/links/link';
+import {tn} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {AvatarProject} from 'sentry/types/project';
+
+export type Monitor = {
+  id: string;
+  name: string;
+  project: AvatarProject;
+  description?: string;
+};
+
+type MonitorsCellProps = {
+  monitors: Monitor[];
+};
+
+export function MonitorsCell({monitors}: MonitorsCellProps) {
+  if (monitors.length === 0) {
+    return <div>&mdash;</div>;
+  }
+  return (
+    <div>
+      <Hovercard
+        body={monitors.map(({name, id, project, description}, index) => (
+          <Fragment key={id}>
+            {index > 0 && <Divider />}
+            <HovercardRow to={`/monitors/${id}/`}>
+              <strong>{name}</strong>
+              <MonitorDetails>
+                <ProjectBadge
+                  css={css`
+                    && img {
+                      box-shadow: none;
+                    }
+                  `}
+                  project={project}
+                  avatarSize={16}
+                  disableLink
+                />
+                {description && (
+                  <Fragment>
+                    <Separator />
+                    {description}
+                  </Fragment>
+                )}
+              </MonitorDetails>
+            </HovercardRow>
+          </Fragment>
+        ))}
+      >
+        <MonitorCount>{tn('%s monitor', '%s monitors', monitors.length)}</MonitorCount>
+      </Hovercard>
+    </div>
+  );
+}
+
+const MonitorDetails = styled('div')`
+  display: inline-grid;
+  grid-auto-flow: column dense;
+  gap: ${space(0.75)};
+  justify-content: start;
+  align-items: center;
+  color: ${p => p.theme.subText};
+  font-size: ${p => p.theme.fontSizeSmall};
+  min-width: 500px;
+  white-space: nowrap;
+  line-height: 1.2;
+
+  @media (min-width: ${p => p.theme.breakpoints.xlarge}) {
+    line-height: 1;
+  }
+`;
+
+const Separator = styled('span')`
+  height: 10px;
+  width: 1px;
+  background-color: ${p => p.theme.innerBorder};
+  border-radius: 1px;
+`;
+
+const Divider = styled('hr')`
```

Comment:
is there a shared component for this already?

Reply:
Surprisingly not! I'd just leave this as a one-off for now. 

We should tackle this as part of the design system. Every `'hr'` in the codebase currently has slightly different margins/styling.